stdlib/public/Concurrency/Task.swift

```diff
@@ -611,6 +611,11 @@ extension Task where Success == Never, Failure == Never {
   public static var name: String? {
     return _getCurrentTaskNameString()
   }
+
+  @available(SwiftStdlib 6.2, *)
+  public var name: String? {
+    return _getCurrentTaskNameString()
```

Comment:
This shouldn't call `getCurrentTaskName` but `_getTaskName` right?

Reply:
thanks yep :) also missed to push tests, added now 😅